test/cpp/tensorexpr/test_bufliverange.cpp

```diff
@@ -1,87 +0,0 @@
-#include <gtest/gtest.h>
-
-#include <stdexcept>
-#include "test/cpp/tensorexpr/test_base.h"
-
-#include <torch/csrc/jit/tensorexpr/analysis.h>
-#include <torch/csrc/jit/tensorexpr/expr.h>
-#include <torch/csrc/jit/tensorexpr/ir.h>
```

Comment:
Why are we removing this test after we added it just a couple commits earlier?

Reply:
Oh, the test is moved to file `test_memplanning.cpp`. We only use `BufLiveRange` in memory planning so it makes more sense to put it there.

Reply:
Code moves likes this make it difficult to review since they camouflage changes in the file, please try to avoid this in future. When in the same stack, why not to put the tests into this file from the beginning, and when it's for existing code, then it's better to move code in a separate PR without other changes.